tests/datavolume_test.go

```diff
@@ -1254,6 +1276,30 @@ var _ = Describe("[vendor:cnv-qe@redhat.com][level:component]DataVolume tests",
 					Message: "Import Complete",
 					Reason:  "Completed",
 				}}),
+			Entry("[test_id:5083]succeed importing VDDK data volume with extra arguments ConfigMap set", dataVolumeTestArguments{
```

Comment:
Do you want to assert something in the final step of this test? like checking the config was applied?

Reply:
For this test I wanted to make sure that the contents of the ConfigMap are present in the file, so the assertion happens in the vddk-test-plugin (the fgets/strcmp). Is there a better way to check the result from the importer? Like can this test read the pod logs or termination message?

Reply:
oh okay I see https://github.com/kubevirt/containerized-data-importer/blob/625a9e944e3b75e514ca67e394c3e9931bf9634a/tools/vddk-test/vddk-test-plugin.c#L50 Yeah I think we can come up with a cleaner way,
You can either read pod logs or make this a part of the termination message struct https://github.com/kubevirt/containerized-data-importer/blob/625a9e944e3b75e514ca67e394c3e9931bf9634a/pkg/importer/vddk-datasource_amd64.go#L1011-L1019

Reply:
WDYT?